pkgs/development/interpreters/evcxr/evcxr.nix

```diff
@@ -0,0 +1,62 @@
+{ callPackage
+, fetchFromGitHub
+, pkg-config
+, lib
+, makeWrapper
+, stdenv
+
+, gcc
+, cmake
+, libiconv
+, CoreServices
+, Security
+
+, cargo
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "evcxr";
+  version = "0.15.0";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "evcxr";
+    rev = "v${version}";
+    sha256 = "sha256-s8zM1vxEeJYcRek1rqUmrBfvB2zCAF3iLG8UVA7WABI=";
```

Comment:
Nit: Since we're touching this, could we opportunistically change this property from `sha256` to `hash`?

Reply:
Done